### PR TITLE
#issue-1108 Describe Image Detail for products

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes the properties of the "Image Detail" window for customizing product images displaying.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/product-images-and-video.html

## Additional information

image

### CODE REVIEW

**Review by atwixfirster — APPROVED** [review]

**Comment by atwixfirster** [comment]
Thank you for your contributions, @ZhovnirMaria 

Your changes totally make sense.

File: src/catalog/product-images-and-video.md [hunk 1]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
     - Click the **camera** (![]({% link images/images/btn-camera.png %})) tile, select the image file, and click <span class="btn">Open</span>.
 
 A [placeholder]({% link catalog/product-image-placeholders.md %}) image appears in the catalog until a product image is available.
+
+## Image Detail
+
+When product images are added, there is the possibility to customize their displaying in the **Image Detail** window.
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:26]:
```suggestion
When product images are added, there is the possibility to customize the display in the _Image Detail_ window.
```

File: src/catalog/product-images-and-video.md [hunk 2]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
...
+
+ ![]({% link images/images/image-video.png %}){: .zoom}
+    _Product images_
+
+Set image displaying options for a product:
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:31]:
```suggestion
To set image displaying options for a product:
```

File: src/catalog/product-images-and-video.md [hunk 3]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
...
+
+1. Click on the product image.
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:33]:
```suggestion
1. Click the product image.
```

File: src/catalog/product-images-and-video.md [hunk 4]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
...
+1. Enter the **Alt text**.
+
+1. Choose a **Role** for displaying the image (all roles can be selected for one image):
+
+    - **Swatch**
+    - **Base**
+    - **Small**
+    - **Thumbnail**
+
+All of these roles are selected for the first added image by default.
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:44]:
```suggestion
   All of these roles are selected for the first added image by default.
```

File: src/catalog/product-images-and-video.md [hunk 5]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
...
+
+ ![]({% link images/images/details.png %}){: .zoom}
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:46]:
```suggestion
   ![]({% link images/images/details.png %}){: .zoom}
```

File: src/catalog/product-images-and-video.md [hunk 6]
```diff
@@ -20,3 +20,30 @@ From the _Images and Videos_ section, you can perform basic image management tas
...
+    _Image details_
+
+To hide the product image from the product page select the **Hide from Product Page** checkbox.
```
Comment (leslietilling) [inline src/catalog/product-images-and-video.md:49]:
```suggestion
To hide the product image from the product page, select the **Hide from Product Page** checkbox.
```

**Review by leslietilling — COMMENTED** [review]
Edits for style, grammer, and formatting

**Comment by ZhovnirMaria** [comment]
Thank you @leslietilling, for your updates!

**Comment by ZhovnirMaria** [comment]
Hi, @leslietilling 
I made some changes to the request, you can check them.
Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1110`.

**Pull request merged**